index.js

```diff
@@ -23,8 +23,10 @@ try {
 module.exports = firewall;
 module.exports.forgetKnownHosts = forgetKnownHosts;
 
-function firewall(app) {
+function firewall(passedObject) {
   // https://expressjs.com/en/4x/api.html
+  const app = 'app' in passedObject ? passedObject.app : passedObject;
```

Comment:
Here you've implicitly redefined the meaning of the comment with the link to Express API.

Earlier it looked like this:

```js
// https://expressjs.com/en/4x/api.html
app.use((req, res, next) => {})
```

The comment was related to the `app.use` and the its callback list of params.

Now it looks like this:

```js
// https://expressjs.com/en/4x/api.html
const app = 'app' in passedObject ? passedObject.app : passedObject;
```

Which means that the fact of `app` held in `passedObject` is somehow related to the Express API. But it is not.

What you should probably do is to move the comment below to its original place. Also it would be great to add the comment to the added line. Just to clarify what it does.

Reply:
My bad 🙃 

This comment was returned back and new one added.